src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  let price = cost * days;
```

Comment:
What exactly cost and price is it? Let's specify it.
Also, can't you make it a const and just return price - discount?